src/poliastro/plotting/misc.py

```diff
@@ -15,75 +14,61 @@
     Venus,
 )
 from poliastro.frames import Planes
-from poliastro.plotting.interactive import OrbitPlotter2D, OrbitPlotter3D
-from poliastro.plotting.static import StaticOrbitPlotter
+from poliastro.plotting import OrbitPlotter
 
 
-def _plot_bodies(orbit_plotter, outer=True, epoch=None):
-    bodies = [Mercury, Venus, Earth, Mars]
-    if outer:
-        bodies.extend([Jupiter, Saturn, Uranus, Neptune])
-
-    for body in bodies:
-        orbit_plotter.plot_body_orbit(body, epoch)
-
-
-def _plot_solar_system_2d(epoch, outer=True, interactive=False):
-    if interactive:
-        orbit_plotter = OrbitPlotter2D(
-            plane=Planes.EARTH_ECLIPTIC
-        )  # type: Union[OrbitPlotter2D, StaticOrbitPlotter]
-    else:
-        orbit_plotter = StaticOrbitPlotter(plane=Planes.EARTH_ECLIPTIC)
-
-    orbit_plotter.set_body_frame(Earth, epoch)
-
-    _plot_bodies(orbit_plotter, outer, epoch)
-
-    return orbit_plotter
-
-
-def _plot_solar_system_3d(epoch, outer=True):
-    orbit_plotter = OrbitPlotter3D(plane=Planes.EARTH_ECLIPTIC)
-
-    _plot_bodies(orbit_plotter, outer, epoch)
-
-    return orbit_plotter
-
-
-def plot_solar_system(outer=True, epoch=None, use_3d=False, interactive=False):
+def plot_solar_system(
+    backend_name="matplotlib2D", epoch=None, labels=None, outer=True
```

Comment:
Maybe I'd make the `backend_name` the last parameter, to avoid breaking backwards compatibility?

Reply:
I agree.